### PR TITLE
Pre commit test

### PR DESCRIPTION
**Pull request by mybffjustin**
Prettier format https://prettier.io/docs/en/precommit.html

### CODE REVIEW

**Review by matty377 — COMMENTED** [review]
The /target/ folder should not be checked into git and should be in the .gitignore to prevent future check-ins of those files. /target/ is a generated folder to hold your build output.

**Comment by mybffjustin** [comment]
Thank you for the review🙏🙏🙏
I think I had "target/" in gitignore instead of "/target/" for some reason🤣
I will fix this👍👍

**Comment by trucdg** [comment]
> Thank you for the review🙏🙏🙏 I think I had "target/" in gitignore instead of "/target/" for some reason🤣 I will fix this👍👍

wait a minute, I thought  "target/" is the correct syntax to ignore a folder? I believe that I added "target/" in .gitignore in my previous commits.

image

**Comment by matty377** [comment]
Adding something to .gitignore won't remove it if it's already checked in. You'll have to additionally `git rm` the errant files/folders.

**Comment by mybffjustin** [comment]
Yeah I realized that I checked out someone's branch and it removed my gitignore because the branch checked out didn't have a gitignore so it deleted my gitignore🤣🤣🤣
Because I originally had gitignore the same as Truc

**Comment by mybffjustin** [comment]
That's what I meant in our group chat about untracking target

**Comment by mybffjustin** [comment]
Would you like me to use my original gitignore and push again?

File: group-project/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
   </head>
   <body>
     <h1>Google Map Demo</h1>
 
     <div id="input-container">
-        <span><i class="fa-solid fa-street-view fa-xl" id="location-icon"></i></span>
-        <input type="text" placeholder="Enter your location" id="search-input">
+      <span
+        ><i class="fa-solid fa-street-view fa-xl" id="location-icon"></i
+      ></span>
```
Comment (trucdg) [inline group-project/src/main/webapp/index.html:20]: nit: the alignment and new lines in between the element <span> and <i> tag is kinda weird

Comment (trucdg) [reply group-project/src/main/webapp/index.html:20]: Wait, is it how Prettier formats the code?

**Review by trucdg — APPROVED** [review]
Some formatted element tags look strange to me but I think it's how Prettier formats code. I think it's fine. I tested the branch and .gitignore worked! I think it's good to merge

**Comment by trucdg** [comment]
Btw, I think commit messages can be more detailed. Yet, if they serve the same purpose, and you want to name all of them as "git ignore", I think they can be combined into 1 commit.

**Pull request merged**